### DISCUSSION TITLE
repository personalizzati in HACS

### DISCUSSION THREAD

**Discussion by loregcl** [indomus/forum#1486] [HUB: Home Assistant]
ciao,

ho istallato HA core su Docker e ho installato HACS come integrazione. Funziona tutto, ma se cerco il tasto per aggiungere repository personalizzati non c'è...
Nel menù dei tre puntini ci sono solo 

- informazioni
- repository di Hacs
- aggiorna informazioni
- scarica di nuovo
- problemi irrisolti

da cosa può dipendere?

grazie

**Comment by snakuzzo** [comment]
![aaa](https://user-images.githubusercontent.com/7205371/153242700-928a162c-2533-47c6-a3e2-2e0a685817e0.png)

**Reply by loregcl** [reply]
grazie mille! che scemo, non avevo visto quel menù!
